travis/mvn-settings.xml

```diff
@@ -1,4 +1,12 @@
 <settings>
+    <mirrors>
+        <mirror>
+            <id>maven-central-mirror</id>
+            <name>Maven central mirror</name>
+            <url>https://maven-central.storage-download.googleapis.com/maven2/</url>
+            <mirrorOf>central</mirrorOf>
```

Comment:
Hi, what is this mirror for?

Reply:
That's the default mirror travis uses if we don't overwrite the settings.xml file (you can see that in any travis logs). When I started using this mvn_settings.xml , some builds failed trying to transfer artifacts from maven central.